pom.xml

```diff
@@ -30,7 +30,7 @@
         <spring-shell.version>2.0.0.RELEASE</spring-shell.version>
         <spring-statemachine.version>1.2.8.RELEASE</spring-statemachine.version>
 
-        <spring-cloud.version>Dalston.RELEASE</spring-cloud.version>
+        <spring-cloud.version>Dalston.SR5</spring-cloud.version>
```

Comment:
LGTM, will add a comment here and the parent `spring-cloud-build` entry to make these versions in sync from `spring-cloud-release`.